facets_mcp/tools/deploy_module.py

```
@@ -250,34 +274,41 @@ def check_deployment_status(cluster_id: str, deployment_id: str, wait: bool = Fa
                 "success": deployment.status == "SUCCEEDED",
-                "status": deployment.status,
-                "deployment_id": deployment_id,
-                "cluster_id": cluster_id,
-                "started_at": deployment.created_at.isoformat() if hasattr(deployment,
-                                                                           'created_at') and deployment.created_at else None,
-                "completed_at": deployment.completed_at.isoformat() if hasattr(deployment,
-                                                                               'completed_at') and deployment.completed_at else None,
-                "triggered_by": deployment.triggered_by if hasattr(deployment, 'triggered_by') else None,
                 "message": f"Deployment {deployment.status}",
-                "elapsed_seconds": elapsed_time,
+                "errors" : None if deployment.status == "SUCCEEDED" else f"Deployment ended with status: {deployment.status}",
```

Pylint (3.3.7)

[convention] Line too long (126/100)

(C0301)